processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -480,42 +481,79 @@ public int compare(ByteBuffer lhsBuffer, ByteBuffer rhsBuffer, int lhsPosition,
       }
     }
 
-    @Override
-    public Comparator<Grouper.Entry<RowBasedKey>> entryComparator()
+    private Grouper.KeyComparator getBufferDimComparatorForNullFudgeTimestamp()
```

Comment:
i think that sounds fine... I guess, I'm into too much mood of comparators mode.